resources/configure_dependencies.py

```diff
@@ -662,53 +561,24 @@ def InstallVTK(pkg: str, ver: str, gold_file: str):
     # Create envvars file
     module_file = open(module_file_name, "w")
 
-    lua_version = package_info["lua"][VERSION]
-    petsc_version = package_info["petsc"][VERSION]
-    vtk_version = package_info["vtk"][VERSION]
-
-    petsc_dir = f"{install_dir}"
-    vtk_dir = f"{install_dir}"
+    #lua_version = package_info["lua"][VERSION]
+    #petsc_version = package_info["petsc"][VERSION]
+    #vtk_version = package_info["vtk"][VERSION]
+    #caliper_version = package_info["caliper"][VERSION]
```

Comment:
If this is not needed, then remove those lines...

Reply:
Fixed.